kubespawner/spawner.py

```diff
@@ -1486,44 +1528,95 @@ def _validate_image_pull_secrets(self, proposal):
         - `description`: Optional description of this profile displayed to the user.
         - `kubespawner_override`: a dictionary with overrides to apply to the KubeSpawner
           settings. Each value can be either the final value to change or a callable that
-          take the `KubeSpawner` instance as parameter and return the final value.
+          take the `KubeSpawner` instance as parameter and return the final value. This can
+          be further overridden by 'options'
+        - 'options': A dictionary of sub-options that allow users to further customize the
+          selected profile. Items should have a unique key representing the customization,
```

Comment:
Mentioning that this is a drop-down list interface is relevant to do here I think.

Reply:
Done